app/__init__.py

```diff
@@ -1,8 +1,13 @@
 from flask import Flask
 from .db import db, migrate
 from .models import task, goal
+from app.routes.task_routes import bp as tasks_bp
+from app.routes.goal_routes import bp as goals_bp
 import os
 
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Were you running into an issue that required calling `load_dotenv()` from here? 

We need to call `load_dotenv()` in `conftest.py` to load up variables for the testing flow, but that shouldn't be necessary in production mode if we've configured things the same as we did for the hello-books and solar-system APIs.